internal/ignition/dummy.go

```diff
@@ -33,7 +32,7 @@ func NewDummyGenerator(workDir string, cluster *common.Cluster, s3Client s3wrapp
 }
 
 // Generate creates the expected ignition and related files but with nonsense content
-func (g *dummyGenerator) Generate(ctx context.Context, installConfig []byte, ocsValidatorConfig *ocs.Config) error {
+func (g *dummyGenerator) Generate(_ context.Context, installConfig []byte) error {
```

Comment:
why there is need for context ? Maybe lets delete it?

Reply:
It is required by the other implementation of the same interface (`ignition.go`)